src/NLog/Targets/Wrappers/AsyncTargetWrapper.cs

```diff
@@ -118,7 +118,12 @@ public AsyncTargetWrapper(Target wrappedTarget)
         /// <param name="overflowAction">The action to be taken when the queue overflows.</param>
         public AsyncTargetWrapper(Target wrappedTarget, int queueLimit, AsyncTargetWrapperOverflowAction overflowAction)
         {
-            RequestQueue = new AsyncRequestQueue(10000, AsyncTargetWrapperOverflowAction.Discard);
+#if NET4_5
+            // Net40 ConcurrencyQueue can seem to leak, because it doesn't clear properly on dequeue
```

Comment:
Please add the links to the code, thanks!

Reply:
Fixed